Src/DependencyCollector/Shared/Implementation/ProfilerSqlProcessingBase.cs

```diff
@@ -92,6 +92,24 @@ public object OnEndForOneParameter(object context, object returnValue, object th
         }
 
         /// <summary>
+        /// On end async callback for methods with 1 parameter.
+        /// </summary>
+        public object OnEndAsyncForOneParameter(object context, object returnValue, object thisObj)
```

Comment:
yes

---
In reply to: [98078491](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r98078491) [](ancestors = 98078491)